### PR TITLE
Utility methods

### PR DESCRIPTION
**Pull request by sayakpaul**
@deep-diver I have added a few utility methods in this PR. I wanted to have your reviews on them before I move to create the Flask script. Thanks.

### CODE REVIEW

File: server/pixabay_utils.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+"""
+
+from io import BytesIO
+from PIL import Image
+import logging
+import requests
+import time
+
+ORIGINAL_API = "https://pixabay.com/api/?key="
+API_KEY = ""  # Change it
```
Comment (sayakpaul) [inline server/pixabay_utils.py:13]: @deep-diver if you have some suggestions on how we should tackle this here, please let me know.

Comment (deep-diver) [reply server/pixabay_utils.py:13]: For deploying to app engine, we could make the API key as a GitHub Secret and inject it as an environment variable,

Or we can Encrypt it using GPG, what do you think?

Comment (sayakpaul) [reply server/pixabay_utils.py:13]: GitHub secret sounds good to me. I will send it over to you so that you can add it. Works?

Comment (deep-diver) [reply server/pixabay_utils.py:13]: sure!

Comment (sayakpaul) [reply server/pixabay_utils.py:13]: But how should we retrieve that from GitHub secrets? Will it be available via `os.environ`?

**Pull request merged**